app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+
+import { Formik, Form as FormikForm } from "formik";
+import { Button, Pane } from "neetoui";
+import { Input, Select } from "neetoui/formik";
+
+import formValidationSchemas from "constants/formValidationSchemas";
```

Comment:
What's the point of keeping validation schemas in `constants` folder? I had already given many review for you pointing out to keep constants within the closest scope.